src/pilot/templates/network-isolation.yaml

```diff
@@ -0,0 +1,70 @@
+# Enable the creation of Neutron networks for isolated Overcloud
```

Comment:
Should put the standard licensing head at the top of this file.

Reply:
Done.